editor.planx.uk/src/routes/authenticated.tsx

```diff
@@ -31,6 +31,11 @@ const editorRoutes = compose(
               name
               slug
             }
+            teamThemes: teams_summary {
+              slug
+              primaryColour: primary_colour
+              logo
+            }
```

Comment:
Rather than adding an additional query here, a better pattern (and actually I think the intended one) would be to use the existing relationships to find the corresponding theme details for the relevant team directly in this query (as opposed to having to match them up later using the `.find()` iterator).

Try this out in Hasura - it should be both faster/more efficient and allow us to simplify our code - 

```gql

query GetTeams {
  teams(order_by: {name: asc}) {
    id
    name
    slug
    theme {
      primaryColour: primary_colour
      logo
    }
  }
}
```